template/.github/ISSUE_TEMPLATE/publish_component.md

```diff
@@ -9,47 +9,62 @@ labels: 'Status: Review Needed, not-reviewed'
 
 ## Specific steps required to publish this component.
 
-All of the below outlined steps must be completed.
+1. __Design & Planning__
+    - [ ] Reviewed [AuroLabs documentation](http://auro.alaskaair.com/)
+    - [ ] Reviewed [Auro's Contributing Guidelines](http://auro.alaskaair.com/contributing)
+    - [ ] Reviewed [Auro's CSS conventions](http://auro.alaskaair.com/webcorestylesheets/conventions) and how to apply CSS to custom elements
+    - [ ] Design team engaged in designing the new component
+    - [ ] Design work integrated in the Auro Design System UI Kit
+1. __Development__
+    - [ ] Addresses all design and UX as proposed in the design comp
+    - [ ] A11y in regards to role and live notifications need to be coded into the final WC
+    - [ ] All style code follows [Auro's Style Guide](http://auro.alaskaair.com/webcorestylesheets/guidelines)
+    - [ ] Automated tests created
+1. __Documentation__
+    - `demo.md`
+      - [ ] Component properly described
+      - [ ] All component `attributes` listed and described
+      - [ ] Component use case(s) outlined
+      - [ ] `Do Not` section outlines implementation scenarios to avoid
+      - Example Section
+        - [ ] Basic example of simplest use case
+        - [ ] Examples showing use of each API attribute
+        - [ ] All examples include rendered example and matching code sample
+    - `readme.md`
+      - [ ] Component properly described
+      - [ ] Bundled version info
+      - [ ] Use cases described
+      - [ ] `API Code Examples` updated to match API with working example code
```

Comment:
For maintenance, I am really leaning toward not putting too many examples in the README. As things change, this is always forgotten. 

I'd much prefer that we link to the Docsite whenever possible for stuff like this.

Reply:
The readme.md template has a single code example. This checkbox is to make sure that example is valid within the API definition.

If we want to change the current template content let's open a separate ticket for that. I would be in favor the readme.api simply referencing/linking the demo examples via a link and remove the code example.